### ISSUE TITLE
[QoL] Add the refresh button to source screens blocked by Cloudflare

### ISSUE THREAD

**Issue by Palloxin** [LNReader/lnreader#667]
### Describe your suggested feature

![IMG_20230619_195647](https://github.com/LNReader/lnreader/assets/75091899/8d5f42b1-9064-4ffc-8d8d-bb194fe24dcf)
After i open WebView, bypass Cloudflare and then come back i would like a refresh button because to make the bypassing of Cloudflare effective i have to exit the source and click on it again.

It should be like this:
![IMG_20230619_200010](https://github.com/LNReader/lnreader/assets/75091899/da685128-6bfb-4cdf-b131-153ae9fe290c)

### Other details

EDIT:
So the same happens with this error page. 
![IMG_20231111_110434](https://github.com/LNReader/lnreader/assets/75091899/8d232cc3-8257-4c6b-affd-6c9c0f0ae974)

Make sure it is applied here too.
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.14](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
